react/components/ProductCustomizer.js

```diff
@@ -212,6 +212,14 @@ class ProductCustomizer extends Component {
       },
     })
 
+    this.scrollToIngredients()
+  }
+
+  scrollToIngredients = () => {
+    // TODO: find a better way for getting the top menu element
+    const topbar = document && document.querySelector('.vtex-top-menu') 
```

Comment:
The "right" way to do this would be for `dreamstore-header` to have a ref and that available to be read by other apps. But, as of today, I don't know of any simple way for apps to communicate with each other or execute methods, because AFAIK the Render Framework has no baked in way of doing something like that.